Supplemental data public docs (#477)
codatio/codat-docs: docs/using-the-api/supplemental-data/usecases.md

---
title: "Common uses of supplemental data"
sidebar_label: "Examples"
description: "Review properties commonly used by our customers to enhance data required for their use cases"
---

> When a particular integration contains a property which is not common across other platforms, it may happen that it does not form part of our standard data, but can be obtained using supplemental data if it benefits your use case. 

Reviewer: ```suggestion
When a particular integration contains a property which is not common across other platforms, it may happen that it does not form part of our standard data, but can be obtained using supplemental data to support your use case. 

```
